test/blackbox/restore_s3_blobs.bats

```diff
@@ -124,10 +125,12 @@ function teardown_file() {
 
     # start with dedupe disabled
     zot_serve ${zot_config_file_nodedupe}
-    wait_zot_reachable 8080
```

Comment:
no need to wait, just read the log file

Reply:
wait_for_string uses tail to catch the log message, if we wait for zot to be reachable startup dedupe may have finished in the meantime. 

I moved the wait_zot_reachable in the second test

Reply:
Looks good.